.npmrc

```diff
@@ -0,0 +1,14 @@
+hoist-pattern[]=!**/typechain/*
+hoist-pattern[]=!**/@typechain/*
+hoist-pattern[]=!@eth-optimism/contracts-bedrock/ds-test
+hoist-pattern[]=!@eth-optimism/contracts-bedrock/forge-std
+hoist-pattern[]=!@eth-optimism/contracts-bedrock/@rari-capital/solmate
+hoist-pattern[]=!@openzeppelin
+hoist-pattern[]=!@eth-optimism/contracts-periphery/ds-test
+hoist-pattern[]=!@eth-optimism/contracts-periphery/forge-std
+hoist-pattern[]=!@eth-optimism/contracts-periphery/@rari-capital/solmate
+hoist-pattern[]=!forta-agent
```

Comment:
```suggestion
``` 
Haven't tested yet but these are all unnecessary.  By default, pnpm never hoists.   This makes a lot of issues I've seen @maurelian and @smartcontracts battle with from time to time go away.

Reply:
I didn't even know this file existed.

Reply:
It didn't exist until this pr.   What I meant was the behavior pnpm has of never hoisting by default is what fixes a lot of pain you and Kelvin went through.  Anytime a type stopped working randomly or some dependency changed out from under you part of what caused that con fusing behavior was how yarn/npm hoists.